notebooks/MosvizNIRISSExample.ipynb

```diff
@@ -113,14 +114,252 @@
    "id": "1c75b8a1",
    "metadata": {},
    "outputs": [],
+   "source": [
+    "%load_ext snakeviz\n",
+    "\n",
+    "%snakeviz mosviz.load_niriss_data(data_dir)"
```

Comment:
Did you mean to leave the profiler in?

Reply:
Frankly I don't think I meant to include either of the notebooks at all. I'll remove them.

Reply:
If you remove any files, please squash them out of commit history too. Thanks!